CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -466,7 +479,7 @@ class CameraFragment : Fragment() {
          * automatically closed after this method returns
          * @return the image analysis result
          */
-        override fun analyze(image: ImageProxy, rotationDegrees: Int) {
+        override fun analyze(@NonNull image: ImageProxy) {
```

Comment:
The annotation is unnecessary in Kotlin, please remove -- otherwise it would be `ImageProxy?`

Reply:
That's indeed useless in Kotlin.